client/app/bundles/course/survey/pages/ResponseIndex/index.jsx

```diff
@@ -71,18 +97,36 @@ class ResponseIndex extends React.Component {
     }).isRequired,
   };
 
+  static computeStatuses(responses) {
```

Comment:
Yes, one of the motivations is performance since only one copy of the static variable is instantiated, though the improvement might not be big. There is a [lint rule](http://eslint.org/docs/rules/class-methods-use-this) for this.